static/scss/pages/settings.scss

```
@@ -0,0 +1,125 @@
+.settings-main {
+
+    .c-settings-headline {
+        @include text-title-2xs;
+        padding: $spacing-lg $spacing-lg $spacing-md;
+        color: $color-grey-60;
+    }
+
+    .c-meta--links {
+        display: none;
+    }
+
+    .c-settings-form--heading {
+        @include text-title-3xs;
+        color: $color-grey-50;
+        font-weight: normal;
+    }
+
+    .c-settings-form--footer {
+        background-color: transparent;
+        text-align: center;
+        display: block;
+    }
+
+    .c-settings-form--field {
+        border-bottom: 1px solid $color-grey-10;
+    }
+
+    .c-settings-form--inline-warning {
+        box-shadow: none;
+        text-align: start;
+        margin-left: 0;
+    }
+
+    .c-settings-form--submit {
+        padding: $spacing-sm $spacing-2xl;
+    }
+
+    .c-settings-form--notification {
+        box-shadow: var(--dropShadowLight);
+        background-color: $color-white;
+        margin: $spacing-md;
+        padding: $spacing-md;
+        border-radius: $border-radius-md;
+    }
+
+    .c-settings-form--notification-heading {
+        @include text-title-3xs;
+        color: $color-grey-60;
+        border-left-width: 4px;
+        border-left-style: solid;
+        margin: 0;
+        padding: $spacing-sm $spacing-md;
+
+        &.t-error {
+            border-color: $color-error;
+        }
+
+        &.t-warning {
+            border-color: $color-warning;
+        }
+    }
+
+    .c-settings-form--notification-description {
+        border-left-width: 5px;
```

Can we use something  `4px` here? We're in a pseudo–base 8 format for CSS units. (With `4` being a `1/2` of [8](url)) 
```suggestion
        border-left-width: 4px;
```

---

https://github.com/mozilla/fx-private-relay/pull/1180/commits/0174cefcbe4edf8ef5e4f88c9261675a3c91c4f3